### PR TITLE
Paths handling updates

### PR DESCRIPTION
**Pull request by MiranDMC**
updated project settings
plugins moved to cleo\cleo_plugins directory
rewritten Current Working Directory handling
introduced 'virtual absolute paths'
addded CLEO_ResolvePath export

Solves https://github.com/cleolibrary/CLEO4/issues/99
Solves #98

### CODE REVIEW

**Comment by x87** [comment]
Please rebase this branch

File: source/CleoVersion.h [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#define CLEO_VERSION_MAIN    4
+#define CLEO_VERSION_MAJOR   4
+#define CLEO_VERSION_MINOR   5
```
Comment (x87) [inline source/CleoVersion.h:5]: please bump to 4.5.0

File: source/CCustomOpcodeSystem.cpp [hunk 2]
```diff
@@ -1706,22 +1701,28 @@ namespace CLEO {
...
+			std::string_view strExport = str.substr(0, pos);
+			std::string_view strModule = str.substr(pos + 1);
 
 			// get module's absolute path
-			std::string modulePath(&str[pos + 1]);
-			modulePath = ResolvePath(modulePath.c_str(), thread->GetScriptFileDir());
+			std::string modulePath;
+			if (strModule.length() < 2 || strModule[1] != ':') // not starting with drive letter
+			{
+				modulePath = "2:\\"; // current script's location
```
Comment (x87) [inline source/CCustomOpcodeSystem.cpp:1720]: please make constants for new virtual drives (0:\\, 1:\\, etc)

File: source/CleoVersion.h (hunk 1 above)
Comment (MiranDMC) [reply source/CleoVersion.h]: Done

File: source/CCustomOpcodeSystem.cpp (hunk 2 above)
Comment (MiranDMC) [reply source/CCustomOpcodeSystem.cpp]: Done

**Pull request merged**